ai_guidelines.md

```diff
@@ -0,0 +1,136 @@
+# Guidelines for CommCare HQ
+
+This document outlines recommended practices for working with the CommCare HQ
+codebase. These guidelines apply in the following circumstances:
+
+- Reviewing code or suggesting changes
+- Writing code and making changes
+- Assisting with version control
+
+## General Principles
+
+- Focus on clarity, maintainability, and performance
+- Consider both immediate needs and long-term implications
+
+## Communication
+
+- The AI Assistant should ask clarifying questions whenever details are unclear,
+  or choices are not obvious.
+- When appropriate, the AI Assistant should make recommendations on how the
+  Developer can improve their prompt to get a more effective response.
+
+## Tech Stack
+
+- Backend: Python, Django
+- Python dependency management: uv
+- Testing: pytest
+- Linting & formatting: Flake8, isort, YAPF
+- Frontend: JavaScript, HTMX, Alpine.js, Knockout.js (legacy), Bootstrap
```

Comment:
should we be specific that some pages are still on Bootstrap 3 and we are in the process of migrating from Bootstrap 3 to 5?